src/components/MovieCard/MovieCard.tsx

```diff
@@ -3,44 +3,55 @@ import './MovieCard.scss';
 
 interface Props {
   movie: Movie;
+  value: string;
 }
 
-export const MovieCard: React.FC<Props> = ({ movie }) => {
+export const MovieCard: React.FC<Props> = ({ movie, value }) => {
   const {
     imdbUrl, imgUrl, description, title,
   } = movie;
 
-  return (
-    <div className="card">
-      <div className="card-image">
-        <figure className="image is-4by3">
-          <img
-            src={imgUrl}
-            alt="Film logo"
-          />
-        </figure>
-      </div>
-      <div className="card-content">
-        <div className="media">
-          <div className="media-left">
-            <figure className="image is-48x48">
-              <img
-                src="images/imdb-logo.jpeg"
-                alt="imdb"
-              />
-            </figure>
-          </div>
-          <div className="media-content">
-            <p className="title is-8">{title}</p>
-          </div>
+  const valueTrimedLowered = value.trim().toLocaleLowerCase();
```

Comment:
probably better this way?
```suggestion
  const finalQuery = value.trim().toLocaleLowerCase();
```